docs/writing-stories/decorators.md

```diff
@@ -155,3 +155,7 @@ All decorators relevant to a story will run in the following order once the stor
 - Global decorators, in the order they are defined
 - Component decorators, in the order they are defined
 - Story decorators, in the order they are defined
+
+## Multiple decorators
+
+You can define multiple decorators, but keep in mind that the last one in the list will wrap the story first, then the earlier ones will wrap those.
```

Comment:
@glrodasz If you're ok with it, I'd suggest a bit more explaining and if possible an example to demonstrate this behavior, to allow our community to have a bit more context on it What do you think?

Reply:
Sure! give me this week and I will improve it.